connectivity/check/test.go

```diff
@@ -11,9 +11,18 @@ import (
 	"sync"
 	"time"
 
+	"github.com/cilium/cilium-cli/defaults"
+	k8sConst "github.com/cilium/cilium/pkg/k8s/apis/cilium.io"
 	ciliumv2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
```

Comment:
Since `cilium/cilium` is considered 3rd party in this repo, there should be a newline separating the local imports (`cilium/ciluim-cli` pkgs)

```suggestion
	"github.com/cilium/cilium-cli/defaults"
	
	k8sConst "github.com/cilium/cilium/pkg/k8s/apis/cilium.io"
	ciliumv2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
```